app/i18n/es/gen.php

```diff
@@ -128,6 +128,7 @@
 		'cz' => 'Čeština',
 		'de' => 'Deutsch',
 		'en' => 'English',
+		'en-us' => 'English (United States)',	// TODO - Translation
```

Comment:
This should be ignored as we wont translate it. The same applies for all supported languages.

Reply:
Agree, revised